src/Look/TrailLook.cpp

```diff
@@ -54,6 +54,20 @@ GetVario2Color(short ramp_h) {
                          ARRAY_SIZE(snail_colors_vario2));
 }
 
+static RGB8Color
+GetVarioKoboColor(short ramp_h) {
+  static constexpr ColorRamp snail_colors_vario_kobo[] = {
+    {  0, {0x00, 0x00, 0x00}},
+    { 50, {0x20, 0x20, 0x20}},
+    {100, {0x40, 0x40, 0x40}},
+    {150, {0x60, 0x60, 0x60}},
+    {200, {0x80, 0x80, 0x80}}
```

Comment:
Why are there 5 items in the array - wouldn't just 2 items result in the same color ramp?

Reply:
True, I started by tweaking another color ramp and lost contact... Now it's fixed.